api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -574,7 +565,7 @@ public static Module getModuleByPackage(String modulePackage) {
 	 * @see Daemon#startModule(Module)
 	 */
 	public static Module startModule(Module module) throws ModuleException {
-		return Daemon.startModule(module);
```

Comment:
i thought this [line](https://github.com/ivange94/openmrs-core/blob/TRUNK-4668/api/src/main/java/org/openmrs/module/ModuleFactory.java#L612) is enough to start in a demon thread?